code/modules/mob/living/simple_animal/borer.dm

```diff
@@ -150,6 +150,33 @@ var/global/borer_unlock_types_leg = typesof(/datum/unlockable/borer/leg) - /datu
 				if(prob(host.brainloss/20))
 					host.say("*[pick(list("blink","blink_r","choke","aflap","drool","twitch","twitch_s","gasp"))]")
 
+	if(client)
+		var/severity = 0
+		switch(health)
+			if(20 to INFINITY)
```

Comment:
You should do these as segments of max_health. You also don't set severity to 1 in this block, or if it's meant to be 0, severity is offset by 1 for all the other values.